jsaddle/js/jsaddle-core.js

```diff
@@ -123,9 +125,33 @@ function jsaddle(global, sendRsp, processSyncCommand) {
   var wrapVal = function(val) {
     return wrapValWithDefault(val);
   };
+  var doSendRsp = function () {
+    if (responses.length > 0) {
+      sendRsp(responses);
+      responses = [];
+    }
+  };
+  var appendRsp = function(rsp) {
+    responses.push(rsp);
+    if (responses.length >= RESPONSE_BUFFER_MAX_SIZE) {
+      doSendRsp();
+    } else {
+      if (sendRspScheduled === false) {
+        sendRspScheduled = true;
+        setTimeout(function() {
+          sendRspScheduled = false;
+          doSendRsp();
+        }, 1);
```

Comment:
Note that you can do `setTimeout(f, 0)` which may be closer to what you want

Reply:
@matthewbauer  Added a comment describing why I used 1 here.